codecov_cli/commands/base_picking.py

```diff
@@ -61,7 +61,7 @@ def pr_base_picking(
             extra_log_attributes=dict(
                 pr=pr,
                 slug=slug,
-                token=token,
+                token="NOTOKEN" if not token else (str(token)[:1] + 18 * "*"),
```

Comment:
can we move this to a util function? Or even put it in the `logging_util` that makes the formatting for the logs. We can check if any of the keys is token and hide it there.

Reply:
good point